bin/ns_unshared

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'English'
+require 'fileutils'
+
+def run(*args)
+  Process.wait spawn(*args)
+  raise $CHILD_STATUS.to_s unless $CHILD_STATUS.success?
+end
+
+e_write = IO.new(ARGV[0].to_i)
+r_read = IO.new(ARGV[1].to_i)
+e_write.write('1')
+e_write.flush
+r_read.read(1)
+r_read.close
+ENV['AUTOHCK_SLIRP'] = File.join(ARGV[2], 'sock')
+resolv = File.join(ARGV[2], 'resolv.conf')
+IO.write resolv, 'nameserver 10.0.2.3'
```

Comment:
What is this IP? Should it be hard coded?

Reply:
It is defined by slirp4netns. See: https://github.com/rootless-containers/slirp4netns/blob/master/slirp4netns.1.md#description